HW03_waiy.py

```diff
@@ -0,0 +1,220 @@
+# TODO Do not edit this file directly! Instead, create a new file called
+# TODO a03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Yin Kyay
+# Username: waiy
+#
+# Assignment: A03 Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1kZhHLMYkN9EUyj44LWp_RjF27UMaWgpc2mObpKcDIuA/edit
+#
+#################################################################################
+# Acknowledgements:
+
+#https://www.colorspire.com/rgb-color-wheel/
+#https://www.webucator.com/article/python-color-constants-module/
+
+#
+#
+#################################################################################
+
+
+
+
+
```

Comment:
Large chunks of extra space (outside of what PEP8 recommends) it typically frowned upon.